Gemfile

```diff
@@ -9,6 +9,7 @@ gem "dotenv"
 gem "memo_wise"
 gem "productive", "0.6.73"
 gem "rake"
+gem "rollbar"
 gem "slack-ruby-client"
```

Comment:
we probably don't need `slack-ruby-client` any more.

Reply:
Removed!